### PR TITLE
NOISSUE- Rename Mainflux to Magistrala

### PR DESCRIPTION
**Pull request by JeffMboya**
Pull request title should be `MF-XXX - description` or `NOISSUE - description` where XXX is ID of issue that this PR relate to.
Please review the [CONTRIBUTING.md](./CONTRIBUTING.md) file for detailed contributing guidelines.

### What does this do?
Update magistrala-cli usage

### Which issue(s) does this PR fix/relate to?
No issue

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -130,18 +130,20 @@ Global Flags:
...
 
 
-Use "mainflux-cli channels [command] --help" for more information about a command.
+Use "cli channels [command] --help" for more information about a command.
 ```
 
 ## Service
 
-### Get Magistrala Things services health check
+Magistrala has two services: users and things.
```
Comment (rodneyosodo) [inline docs/cli.md]: Magistrala doesn't have 2 services. Either reword it or remove it

File: docs/cli.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ Executing `build/mainflux-cli` without any arguments will output help with all a
 
 ```bash
 Usage:
-  mainflux-cli [command]
+  cli [command]
```
Comment (rodneyosodo) [inline docs/cli.md]:
```suggestion
  magistrala-cli [command]
```

Same as all other places

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 2 above)
Comment (JeffMboya) [reply docs/cli.md]: okay

File: docs/cli.md [hunk 3]
```diff
@@ -46,7 +46,7 @@ Flags:
   -C, --contact string         Subscription contact query parameter
   -y, --content-type string    Message content type (default "application/senml+json")
   -e, --email string           User email query parameter
-  -h, --help                   help for mainflux-cli
+  -h, --help                   help for magistrala-cli
```
Comment (rodneyosodo) [inline docs/cli.md]: Update the output from `magistrala-cli`

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: config.toml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+raw_output = ""
+user_token = ""
```
Comment (rodneyosodo) [inline config.toml]: We don't need this file

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Also, rename all occurrences of mainflux to magistrala appropriately

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,19 +1,19 @@
 # Contributing to Magistrala
 
 The following is a set of guidelines to contribute to Magistrala and its libraries, which are
-hosted on the [Magistrala Organization](https://github.com/magistrala) on GitHub.
+hosted on the [Magistrala Organization](https://github.com/mainflux) on GitHub.
```
Comment (drasko) [inline CONTRIBUTING.md]: Nope ;)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -43,7 +43,7 @@ clone your fork repository and configure the remotes:
 ```
 git clone https://github.com/<your-username>/docs.git
 cd docs
-git remote add upstream https://github.com/magistrala/docs.git
+git remote add upstream https://github.com/mainflux/docs.git
```
Comment (drasko) [inline CONTRIBUTING.md]: Nope

**Review by drasko — CHANGES_REQUESTED** [review]
Be careful with this renaming - new project name is Magistrala. Only Gitter currently stayed at Mainflux, and we need to change that as well.

File: CONTRIBUTING.md (hunk 6 above)
Comment (JeffMboya) [reply CONTRIBUTING.md]: Let me double check all the URLs

File: .github/CODEOWNERS [hunk 7]
```diff
@@ -1 +1 @@
-*   @mainflux/maintainers
+*   @magistrala/maintainers
```
Comment (rodneyosodo) [inline .github/CODEOWNERS]: This is not correct have a look at https://github.com/absmach/magistrala

File: .gitignore [hunk 8]
```diff
@@ -1 +1,2 @@
 /site
+debug.log
```
Comment (rodneyosodo) [inline .gitignore]: Remove

File: docs/api.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ API reference in the Swagger UI can be found at: [https://api.magistrala.abstrac
 
 ### Create User
 
-To start working with the Mainflux system, you need to create a user account.
+To start working with the magistrala system, you need to create a user account.
```
Comment (rodneyosodo) [inline docs/api.md]: Capitalize every instance of magistrala except for some rare cases where it is suffixed with something for example `magistrala-vault`

File: docs/certs.md [hunk 10]
```diff
@@ -153,5 +153,5 @@ For more information about the Certification service API, please check out the [
 
 [vault-pki-engine]: https://learn.hashicorp.com/tutorials/vault/pki-engine
 [meodor-vault]: https://github.com/mteodor/vault
-[api-docs]: https://github.com/absmach/magistrala/blob/master/api/certs.yml
-[mainflux-vault]: https://github.com/absmach/magistrala/blob/master/docker/addons/vault/README.md#setup
+[api-docs]: https://github.com/absmach/magistrala/blob/7a2f71a6c1e612e26839eef99f4c99d7f76d9dad/api/openapi/certs.yml#L4
```
Comment (rodneyosodo) [inline docs/certs.md]: This should be from main also remove line number notation

File: docs/edge.md [hunk 11]
```diff
@@ -365,20 +365,20 @@ payload := base64.StdEncoding.EncodeToString(b)
...
 To run it first edit script to set parameters
 
 ```env
 MTLS=false
 EXTERNAL_KEY='raspberry'
 EXTERNAL_ID='pi'
-MAINFLUX_HOST='mainflux.com'
-MAINFLUX_USER_EMAIL='edge@email.com'
-MAINFLUX_USER_PASSWORD='12345678'
+magistrala_HOST='magistrala.com'
```
Comment (rodneyosodo) [inline docs/edge.md]:
```suggestion
MAGISTRALA_HOST='magistrala.com'
```

Same as the surrounding occurrences

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS [hunk 12]
```diff
@@ -1 +1 @@
-*   @mainflux/maintainers
+*   @magistrala/MAINTAINERS
```
Comment (rodneyosodo) [inline .github/CODEOWNERS]: Add a new line

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -13,7 +13,7 @@ Reporting issues are a great way to contribute to the project. We are perpetuall
 thorough bug report.
 
 Before raising a new issue, check [our issue
-list](https://github.com/mainflux/docs/issues) to determine if it already contains the
+list](https://github.com/absmach/magistrala-docs) to determine if it already contains the
```
Comment (rodneyosodo) [inline CONTRIBUTING.md]:
```suggestion
list](https://github.com/absmach/magistrala-docs/issues) to determine if it already contains the
```

File: docs/certs.md [hunk 14]
```diff
@@ -153,5 +153,5 @@ For more information about the Certification service API, please check out the [
 
 [vault-pki-engine]: https://learn.hashicorp.com/tutorials/vault/pki-engine
 [meodor-vault]: https://github.com/mteodor/vault
-[api-docs]: https://github.com/absmach/magistrala/blob/master/api/certs.yml
-[mainflux-vault]: https://github.com/absmach/magistrala/blob/master/docker/addons/vault/README.md#setup
+[api-docs]: https://github.com/absmach/magistrala/blob/main/api/openapi/certs.yml
+[magistrala-vault]: https://github.com/absmach/magistrala/blob/master/docker/addons/vault/README.md#setup
```
Comment (rodneyosodo) [inline docs/certs.md]:
```suggestion
[magistrala-vault]: https://github.com/absmach/magistrala/blob/main/docker/addons/vault/README.md#setup
```

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

File: docs/cli.md [hunk 15]
```diff
@@ -94,62 +99,68 @@ Response should look like this:
...
 ```
 
 ## Service
 
-### Get Magistrala Things services health check
+### Get Magistrala services health check
 
 ```bash
-mainflux-cli health
+magistrala-cli health [service]
```
Comment (rodneyosodo) [inline docs/cli.md]:
```suggestion
magistrala-cli health <service>
```

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by rodneyosodo — DISMISSED** [review]
LGTM

**Review by rodneyosodo — DISMISSED** [review]

**Review by SammyOina — DISMISSED** [review]

File: docs/provision.md [hunk 16]
```diff
@@ -741,18 +741,18 @@ Response contains created things, channels and certificates if any:
...
+git clone https://github.com/absmach/magistrala-ui
+cd magistrala-ui
 docker-compose -f docker/docker-compose.yml up
 ```
 
 Create user and obtain access token
 
 ```bash
-mainflux-cli -m https://mainflux.com users create john.doe@email.com 12345678
+magistrala-cli -m https://mainflux.com users create john.doe@email.com 12345678
```
Comment (dborovcanin) [inline docs/provision.md]: There is no need for `-m` flag here.

File: docs/provision.md [hunk 17]
```diff
@@ -741,18 +741,18 @@ Response contains created things, channels and certificates if any:
...
 
 # Retrieve token
-mainflux-cli -m https://mainflux.com users token john.doe@email.com 12345678
+magistrala-cli -m https://mainflux.com users token john.doe@email.com 12345678
```
Comment (dborovcanin) [inline docs/provision.md]: Same here, remove `-m` and its value.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**